letsencrypt/cli.py

```diff
@@ -631,25 +642,32 @@ def __init__(self, args, plugins):
         #print self.visible_topics
         self.groups = {}  # elements are added by .add_group()
 
-    def preprocess_args(self, args):
-        """Work around some limitations in argparse.
+    def parse_args(self):
+        """Parses command line arguments and returns the result.
+
+        :returns: parsed command line arguments
+        :rtype: argparse.Namespace
 
-        Currently: add the default verb "run" as a default, and ensure that the
-        subcommand / verb comes last.
         """
-        if "-h" in args or "--help" in args:
+        parsed_args = self.parser.parse_args(self.args)
```

Comment:
Nit: can you move the call to `self.determine_verb()` to the line above this without breaking anything?  Since it's part of the parsing process, it makes sense to call here.